node/tests/TestUtilities.ts

```diff
@@ -386,7 +369,7 @@ export const getClientConfigurationOption = (
             port,
         })),
         protocol,
-        useTLS: parseCommandLineArgs()["tls"] == "true",
+        useTLS: global.TLS ?? false,
```

Comment:
```suggestion
        useTLS: !!global.TLS ?? false,
```

Reply:
There's examples in DEVELOPER.md to use tls.  I don't think we need to update those, right?

Reply:
You mean how to use tls? It's not changing usage, it's just a nicer way to arrange code.